inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -26,6 +28,48 @@ def __init__(self, coils, num_field_periods):
         self.num_coils = len(coils)
         self.num_field_periods = num_field_periods
 
+    @classmethod
+    def from_circular_curves(cls, num_field_periods, num_coils, coil_currents,
+                             major_radius, minor_radius):
+        """Create simple circular and equally spaced curves.
+
+        Args:
+            num_field_periods (int): Number of magnetic field periods.
+            num_coils (int): The number of coils per field period.
+            coil_currents (list): List of coil currents.
+            major_radius (float): distance from the center of the torus 
+            (the central axis) to the outer edge of the plasma region.
+            minor_radius (float): Radius of the simple circular curves.
+
+        Returns:
+            StellaratorCoils: The created StellaratorCoils instance.
+        """
+
+        stellarator_symmetry = True
+        angle_factor = (1+int(stellarator_symmetry)) * \
+            num_field_periods*num_coils
```

Comment:
Since we will always want to have `stellarator_symmetry = True`, why don't we simplify this and have `angle_factor = 2 * num_field_periods * num_coils`?

Reply:
Also explain (in a comment) that `angle_factor` is there to spread the coils equally over an angle that is a fraction of 2 $\pi$.